examples/data-block-examples/landsat8-geotiff-example.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "straight-candy",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/wilhelmina.nekoto/.pyenv/versions/UP42-py-dev/lib/python3.7/site-packages/pandas/compat/__init__.py:97: UserWarning: Could not import the lzma module. Your installed Python is incomplete. Attempting to use lzma compression will result in a RuntimeError.\n",
+      "  warnings.warn(msg)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import up42"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "occasional-north",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-15 11:51:31,084 - Authentication with UP42 successful!\n",
+      "2021-03-15 11:51:32,176 - Initialized Project(name: Landsat8-data-example, project_id: fbac95e6-1d78-4953-9485-b6fc072cb4c2, description: , createdAt: 2021-03-15T10:48:54.591911Z)\n"
+     ]
+    }
+   ],
+   "source": [
+    "up42.authenticate(project_id=\"fbac95e6-1d78-4953-9485-b6fc072cb4c2\", \n",
```

Comment:
Please remove those credentials. This is information that needs to kept secure under all circumstances and should never be put into a public facing github repository.
As this information is now already in git, the best solution is to delete this project permanently and use a new one for this activity. Then use the mechanisms you can see in other notebooks how to hide this information.

Reply:
I will proceed to delete the project and create a new one.